packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.types.ts

```diff
@@ -80,6 +80,17 @@ export interface IColorPickerProps {
    * Call to provide customized styling that will layer on top of the variant rules.
    */
   styles?: IStyleFunctionOrObject<IColorPickerStyleProps, IColorPickerStyles>;
+
+  /**
+   * Call to provide customized styling for Preview Box.
+   */
+  rectangleStyle?: IStyleFunctionOrObject<IColorPickerStyleProps, IColorPickerStyles>;
```

Comment:
Can you remove this one? It does not follow any other convention and doesn't seem to be needed.

Reply:
Just catching up on the full conversation here. It seems like the intent is to pass in style overrides to the rectangle. Looking at the code to understand if there's a better way

Reply:
So it seems like the props are wrong anyway here. It probably should be:

```tsx
rectangleStyles: IStyleFunctionOrObject<IColorRectangleStyleProps, IColorRectangleStyles>
```
But, I'd prefer to remove them. Is there a real need here? Can you clarify why?

Reply:
Ok I have removed it now, and I am using colorRectangle Style to pass the height of the rectangle.  Can you now help me merging the PR